fuel-tx/src/transaction/types/script.rs

```diff
@@ -371,6 +371,11 @@ mod field {
                     .zip(input.predicate_len().map(bytes::padded_len_usize))
             })
         }
+
+        #[inline(always)]
+        fn inputs_predicate_gas_used_at(&self, idx: usize) -> Option<Word> {
+            self.inputs().get(idx).and_then(|input| input.predicate_gas_used())
```

Comment:
This is also returning the value instead of the offset.

Reply:
removed the function and refactored